components/studio/bin/hab-studio.sh

```diff
@@ -1041,6 +1049,12 @@ report_env_vars() {
   if [ -n "${no_proxy:-}" ]; then
     info "Exported: no_proxy=$no_proxy"
   fi
+
+  if [ "$(load_secrets)" ]; then
+    for secret in $(load_secrets); do
+      info "$(echo $secret | $bb awk -F= '{print "Exported",$1"=[redacted]"}')"
```

Comment:
Maybe a space after exported like `"Exported "`? Otherwise I think this runs the 2 words together.

Reply:
actually the comma is a space separator so it prints with the space